bite-size-notes/week-6-e-Perception_Aware_MPC_Paper_review.md

```diff
@@ -0,0 +1,107 @@
+
+$$
+\begin{align*}
+C \mathbf{p}_{f}= & \left(\mathbf{q}_{W B} \mathbf{q}_{B C}\right)^{-1} \odot  \tag{4}\\
+& \left({ }_{W} \mathbf{p}_{f}-\left(\mathbf{q}_{W B} \odot \mathbf{p}_{B C}+\mathbf{p}_{W B}\right)\right)
+\end{align*}
+$$
+
+The point $C \mathbf{p}_{f}$ in camera frame is projected into the image plane coordinates $\mathbf{s}=(u, v)^{\top}$ according to classical pinhole camera model :
+
+$$
+\begin{equation*}
+u=f_{x} \frac{C p_{f x}}{C p_{f z}}, \quad v=f_{y} \frac{C p_{f y}}{C p_{f z}} \tag{5}
+\end{equation*}
+$$
+
+where $f_{x}, f_{y}$ are the focal lengths for pixel rows and columns, respectively.
+Later, these equations can be used to express the projection velocity as a function of the quadrotor state and input vectors as we are interested in reducing the velocity of its projection onto the image plane. 
+
+## Problem Formulation
+
+The unified optimization problem can be formulated as following:
+
+$$
+\begin{align*}
+& \min _{\mathbf{u}} \int_{t_{0}}^{t_{f}} \mathcal{L}_{a}(\mathbf{x}, \mathbf{u})+\mathcal{L}_{p}(\mathbf{z}) d t  \tag{1}\\
+& \text { subject to } \quad \mathbf{r}(\mathbf{x}, \mathbf{u}, \mathbf{z})=0 \\
+& \mathbf{h}(\mathbf{x}, \mathbf{u}, \mathbf{z}) \leq 0
+\end{align*}
+$$
+
+Where
+
+$$
+\begin{aligned}
+& \dot{\mathbf{x}}=\boldsymbol{f}(\mathbf{x}, \mathbf{u}) \text { is the system dynamics } \\
+& \mathbf{z}=\boldsymbol{f}_{p}(\mathbf{x}, \mathbf{u}, \sigma) \cdot \text { is the state vector for the perception dynamics } \\
+& \mathcal{L}_{a}(\mathbf{x}, \mathbf{u}) \text { defines an action cost } \\
+& \mathcal{L}_{p}(\mathbf{z}) \text { defines a cost for perception objectives. }
+\end{aligned}
+$$
+
+After deriving the quadrotor dynamics and perception objectives the following cost function was employed in the Model Predictive Control :
+
+$$
+\begin{align*}
+\mathcal{L}= & \overline{\mathbf{x}}_{N}^{\top} \mathcal{Q}_{x, N} \overline{\mathbf{x}}_{N}+ \\
+& +\sum_{i=1}^{N-1}\left(\overline{\mathbf{x}}_{i}^{\top} \mathcal{Q}_{x, i} \overline{\mathbf{x}}_{i}+\overline{\mathbf{z}}_{i}^{\top} \mathcal{Q}_{p, i} \overline{\mathbf{z}}_{i}+\overline{\mathbf{u}}_{i}^{\top} \mathcal{R}_{i} \overline{\mathbf{u}}_{i}\right) \tag{10}
+\end{align*}
+$$
+
+where the values  $\overline{\mathbf{x}}, \overline{\mathbf{z}}, \overline{\mathbf{u}}$ refer to the difference with respect to the reference of each value.
+
+The inputs $\mathbf{u}$, consisting of $c$ and $\boldsymbol{\Omega}_{B}$, as well as the velocity $\mathbf{v}_{W B}$ are limited by the constraints:
+
+$$
+\begin{gather*}
+c_{\min } \leq c \leq c_{\max }  \tag{11}\\
+-\Omega_{\max } \leq \boldsymbol{\Omega}_{B} \leq \Omega_{\max }  \tag{12}\\
+-v_{\max } \leq \mathbf{v}_{W B} \leq v_{\max } \tag{13}
+\end{gather*}
+$$
+
+where $c_{\min }, c_{\max }, \Omega_{\max }, v_{\max } \in \mathcal{R}_{+} \cdot$
+
+## Conclusion
```

Comment:
The writer clearly summarizes the effectiveness of the PAMPC approach and mentions real-world application. The conclusion will be more robust if the writer can also discuss potential limitations, areas for future research, and comparison against traditional MPC approaches.